configs/connection.ts

```diff
@@ -1,26 +1,11 @@
-import * as entities from 'entities'
 import * as bootstrap from './bootstrap'
 import { print } from './utils'
-import { Environment } from './environments'
 import { createConnection } from 'typeorm'
-const mongo = Environment.mongo
 
+const mongoConnect = createConnection().then(() =>
```

Comment:
This function should be rename? (Now we can use multiple databases)

Reply:
True, true